src/lib/plugins/clear.js

```diff
@@ -0,0 +1,186 @@
+module.exports.server = (serv, { version }) => {
+  const mcData = require('minecraft-data')(version)
+  const ChatMessage = require('prismarine-chat')(version)
+  serv.commands.add({
+    base: 'clear',
+    info: 'Clears items from player(s) inventory.',
+    usage: '/clear <player> <item> <maxCount>',
+    parse: /^((?:@|\w)+)?(?: (\w+))?(?: (\d+))?$/, // ex: /clear USERNAME_ stone 1
+    action (params, ctx) {
+      const playersString = (players) => players.length > 1 ? `${players.length} players` : `player ${players[0].username}`
+      const isConsole = Object.entries(ctx).length === 0
+      const isOp = isConsole ? true : ctx.player.op
+      const notUdf = (x) => x !== undefined
+      console.log()
+      if (typeof params === 'string') return `${this.usage}: ${this.info}` // parsing failed
+      if (isConsole && !notUdf(params[1])) return 'A player is required to run this command here' // console
```

Comment:
You can destructure the array to make it easier to follow, like `const [ a, b, c ] = [1, 2]`